Duplicate identifier

See https://github.com/evryfs/sbom-dependency-submission-action/runs/7497968551?check_suite_focus=true - there seems to be a snag with the 1.0.1 release compared to 1.0.0

what i am seeing: 
```text
> typescript-action@0.0.0 build
> tsc

node_modules/@cyclonedx/cyclonedx-library/node_modules/packageurl-js/src/package-url.d.ts(31,9): error TS2300: Duplicate identifier 'PackageURL'.
node_modules/packageurl-js/src/package-url.d.ts(31,9): error TS2300: Duplicate identifier 'PackageURL'.
Error: Process completed with exit code 2.
```

My thoughts: 
Why is there a folder `node_modules/@cyclonedx/cyclonedx-library/node_modules`?  see https://github.com/evryfs/sbom-dependency-submission-action/pull/40#discussion_r930127728
Reason: this is caused by conflicting dependencies on `packageurl-js` in https://github.com/evryfs/sbom-dependency-submission-action/blob/dependabot/npm_and_yarn/cyclonedx/cyclonedx-library-1.0.1/package-lock.json

searched for `packageurl-js` and find 
* see  https://github.com/evryfs/sbom-dependency-submission-action/blob/dependabot/npm_and_yarn/cyclonedx/cyclonedx-library-1.0.1/package-lock.json#L685
* see  https://github.com/evryfs/sbom-dependency-submission-action/blob/dependabot/npm_and_yarn/cyclonedx/cyclonedx-library-1.0.1/package-lock.json#L729

one is requireing ``packageurl-js: ^0.0.7` (meaning range from `>= 0.0.7` to `< 0.0.8`)
the other one is requiring `packageurl-js: 0.0.6` (fixed)

npm resolves the conflict by putting the requrements to their needs.
and then the TypeScript compiler _you_ configured crashes for a reason ... 

Solution i see: have the fixed dependency to `packageurl-js` solved  in `@github/dependency-submission-toolkit`
Best: have the maintainers of `packageurl-js` switch to a proper versioning.

In the meantime i will provide a version of `@cyclonedx/cyclonedx-library` which  downgrades  `packageurl-js: ^0.0.6`  
which again is compatible  with  `packageurl-js: 0.0.6` 
so your issues should be fixed

--- a/HISTORY.md
+++ b/HISTORY.md
@@ -4,12 +4,24 @@ All notable changes to this project will be documented in this file.
 
 ## unreleased
 
+## 1.0.2 - 2022-07-26
+
+Maintenance release.
+
+* Dependencies
+  * Widened the range of requirement `packageurl-js` to `>=0.0.6 <0.0.8`, was `^0.0.7`. ([#130] via [#131])
+
+[#130]: https://github.com/CycloneDX/cyclonedx-javascript-library/issues/130
+[#131]: https://github.com/CycloneDX/cyclonedx-javascript-library/pull/131
+
 ## 1.0.1 - 2022-07-23
 
 Maintenance release.
 
 * Misc
   * Use TypeScript `v4.7.4` now, was `v4.6.4`.
+* Dependencies
+  * Raised the requirement of `packageurl-js` to `^0.0.7`, was `^0.0.6`.
 
 ## 1.0.0 - 2022-06-20
 

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "node": ">=14.0.0"
   },
   "dependencies": {
-    "packageurl-js": "^0.0.7"
+    "packageurl-js": ">=0.0.6 <0.0.8"
   },
   "optionalDependencies": {
     "xmlbuilder2": "^3.0.2"

